traitsui/qt4/tabular_editor.py

```diff
@@ -353,8 +353,10 @@ def _selected_changed(self, new):
             try:
                 selected_row = self.value.index(new)
             except Exception as e:
-                from traitsui.api import raise_to_debug
-                raise_to_debug()
+                # It's not exceptional for new to be None
+                if new is not None:
```

Comment:
Okay, makes sense. I think if I'm going to make a change that alters behaviour in the non-TRAITS_DEBUG case, I need tests.

I'll look into fixing this and adding a test, but it might not happen soon.